spectrum/templates/base.html

```diff
@@ -16,7 +16,11 @@
     <link rel="stylesheet" href="{% static 'info/css/global.css' %}">
     <link rel="stylesheet" href="{% static 'info/css/events.css' %}">
     <script src="https://kit.fontawesome.com/0b4fbfd8b7.js" crossorigin="anonymous"></script>
-    {% block stylesheets %}{% endblock %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>           
```

Comment:
jQuery comes included with django, there's no need to add it.

Reply:
but i was not getting any updates when i changed schedular, after adding that line. It was rendering updates.